client/src/app/core/core-services/autoupdate.service.ts

```diff
@@ -159,7 +184,7 @@ export class AutoupdateService {
             await this.DS.addOrUpdate(changedModels[collection]);
         }
 
-        this.DSUpdateManager.commit(updateSlot);
+        this.DSUpdateManager.commit(updateSlot, fullListUpdateForCollection);
```

Comment:
do not pass this to the DS update manager - it does not have the right responsibility. Just get the whole collections here and do a `DS.remove` for these items (`remove` accepts multiple ids for one collection). So you do not have to modify the update manager and repos.

Reply:
Has the AU-service knowledge about the current view-models in any repo?
I passed these information down to the repos, because I didn't see, that the AU-service have this knowledge and I don't think, that it gets these information.

Reply:
No, it only operates on the DS, not the view model layer. The autoupdate system etc. is only on the model layer, the DSUpdateManager is the link to the view model side. First, everything is injected into the DS, so the DS has a correct and consistent data bases, than the commit is triggered and the repos do their stuff to create view models and publish them.

Reply:
You can either use `DS.getCollection` or write a similar method to `clearMeetingModels` that takes a collection and clears it (Without triggering the clear observable - just doing a `DS.remove` since this method is tied to the DSUpdateManager.